src/03-deals/deals.apib

```diff
@@ -91,6 +91,11 @@ Get a list of deals.
                 + created_at: `2017-05-09T11:25:11+00:00` (string)
                 + updated_at: `2017-05-09T11:30:58+00:00` (string)
                 + `web_url`: `https://app.teamleader.eu/sale_detail.php?id=6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
+                + lost_reason (object)
+                    + reason (object)
+                        + type: lost_reason (string)
```

Comment:
B: types are camel cased, so this should be `lostReason`.

IIRC this is also a requirement to have the lost reasons sideloaded.